src/target/stm32f4.c

```diff
@@ -355,3 +370,30 @@ static bool stm32f4_cmd_option(target *t, int argc, char *argv[])
 	}
 	return true;
 }
+
+static bool stm32f4_cmd_psize(target *t, int argc, char *argv[])
+{
+	if (argc == 1)
+	{
+		uint8_t psize = ((struct stm32f4_flash *)t->flash)->psize;
+		tc_printf(t, "Flash write parallelism: %s\n",
+		          psize == 32 ? "x32" : "x8");
+	}
+	else
+	{
+		uint8_t psize;
+		if (!strcmp(argv[1], "x8"))
+			psize = 8;
+		else if (!strcmp(argv[1], "x32"))
+			psize = 32;
+		else
+		{
+			tc_printf(t, "usage: monitor psize (x8|x32)\n");
+			return false;
+		}
+		for (struct target_flash *f = t->flash; f; f = f->next) {
+			((struct stm32f4_flash *)f)->psize = psize;
```

Comment:
I'm happy to leave it here for now.  This is better than a global or static.

If we support external flashes in future, I think the solution would be to have some signature in the `target_flash` structure so that flashes belonging to this driver can be identified.